lisa/util/shell.py

```diff
@@ -253,6 +254,31 @@ def is_connected(self) -> bool:
             is_inner_shell_ready = True
         return is_inner_shell_ready
 
+    def set_max_connected_sessions_count(self) -> None:
```

Comment:
`set_max_session(self, count:int) -> None:`

Reply:
I will update it in case level.

Reply:
updated